### PR TITLE
Fix: SubsiteVirtualPage will use subsite template #235

### PR DESCRIPTION
**Pull request by torleif**
This is a potential fix for #235

### CODE REVIEW

**Comment by tractorcow** [comment]
Any chance of a unit test?

**Comment by robbieaverill** [comment]
ping @torleif

**Comment by torleif** [comment]
@robbieaverill I've delegated this task to another web dev... expect an updated PR sometime this week

File: code/SubsitesVirtualPage.php [hunk 1]
```diff
@@ -230,4 +230,12 @@ public function init()
         
         Subsite::$disable_subsite_filter = $origDisableSubsiteFilter;
     }
+    
+    
+    public function getViewer($action)
+    {
+        $controller = ModelAsController::controller_for($this->CopyContentFrom());
+        $controller->init();
```
Comment (dhensby) [inline code/SubsitesVirtualPage.php]: I'd be weary of calling `init` - we don't do this on `VirtualPage` (https://github.com/silverstripe/silverstripe-cms/blob/3.5/code/model/VirtualPage.php#L498-L502)

**Comment by robbieaverill** [comment]
Since this is so old now I'm going to close it. I'm not sure why overloading this method and redefining pretty much the same logic would fix it, but hopefully this PR will serve as a useful resource for whoever eventually picks up #235 anyway.

Thanks again for contributing @torleif! Feel free to reopen or recreate this PR if you'd like to revisit it!

**Pull request closed without merging**